tests/Console/ModelsCommand/CustomDate/__snapshots__/Test__test__1.php

```diff
@@ -12,8 +12,8 @@
  * @method static \Illuminate\Database\Eloquent\Builder|CustomDate newModelQuery()
  * @method static \Illuminate\Database\Eloquent\Builder|CustomDate newQuery()
  * @method static \Illuminate\Database\Eloquent\Builder|CustomDate query()
- * @method static \Illuminate\Database\Eloquent\Builder|CustomDate whereCreatedAt($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CustomDate whereUpdatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder|CustomDate whereCreatedAt(\Carbon\CarbonImmutable $value)
+ * @method static \Illuminate\Database\Eloquent\Builder|CustomDate whereUpdatedAt(\Carbon\CarbonImmutable $value)
```

Comment:
Doesn't look correct to me though?

It might be the case that this works, but we also know for a fact you can pass a string.

Previously there was no type so everything was accepted. No it documents it as _only_ objects are accepted.